AsyncDisplayKit/ASVideoNode.mm

```diff
@@ -9,6 +9,23 @@
 #import "ASVideoNode.h"
 #import "ASDefaultPlayButton.h"
 
+static BOOL ASAssetIsEqual(AVAsset *asset1, AVAsset *asset2) {
+  return ASObjectIsEqual(asset1, asset2)
+  || ([asset1 isKindOfClass:[AVURLAsset class]]
+      && [asset2 isKindOfClass:[AVURLAsset class]]
+      && ASObjectIsEqual(((AVURLAsset *)asset1).URL, ((AVURLAsset *)asset2).URL));
+}
+
+static UIViewContentMode ASContentModeFromVideoGravity(NSString *videoGravity) {
+  if ([videoGravity isEqualToString:AVLayerVideoGravityResizeAspect]) {
+    return UIViewContentModeScaleAspectFit;
+  } else if ([videoGravity isEqual:AVLayerVideoGravityResizeAspectFill]) {
+    return UIViewContentModeScaleAspectFill;
+  } else {
+    return UIViewContentModeScaleToFill;
```

Comment:
Is this a good default?  Let's make sure that videoGravity itself defaults to one of the other two options, as stretching a video is almost never intentional / desired.